gcloud-java-dns/src/test/java/com/google/gcloud/dns/ZoneInfoTest.java

```diff
@@ -41,25 +41,23 @@
   private static final String NS2 = "name server 2";
   private static final String NS3 = "name server 3";
   private static final List<String> NAME_SERVERS = ImmutableList.of(NS1, NS2, NS3);
-  private static final ZoneInfo INFO = ZoneInfo.builder(NAME)
+  private static final ZoneInfo INFO = ZoneInfo.of(NAME, DNS_NAME, DESCRIPTION).toBuilder()
       .creationTimeMillis(CREATION_TIME_MILLIS)
       .id(ID)
-      .dnsName(DNS_NAME)
-      .description(DESCRIPTION)
       .nameServerSet(NAME_SERVER_SET)
       .nameServers(NAME_SERVERS)
       .build();
 
   @Test
   public void testDefaultBuilders() {
-    ZoneInfo zone = ZoneInfo.builder(NAME).build();
+    ZoneInfo zone = ZoneInfo.of(NAME, DNS_NAME, DESCRIPTION);
```

Comment:
I renamed the test.

I am using a different object for checking that the read-only fields `id`, `creationTimeMillis` and `nameServerSet` are not initialized. `INFO` has them initialized to some values. I renamed `zone` to `partial` so that this is more obvious.